library/LuaApi.cpp

```diff
@@ -1932,6 +1939,7 @@ static const luaL_Reg dfhack_maps_funcs[] = {
     { "getTileFlags", maps_getTileFlags },
     { "getRegionBiome", maps_getRegionBiome },
     { "getTileBiomeRgn", maps_getTileBiomeRgn },
+    { "getPlantAtCoords", maps_getPlantAtCoords },
```

Comment:
Minor consistency thing: we don't really seem to use `Coords` in any other function names (though I know it was suggested as part of reverse-engineering on the forums). Perhaps `getPlantAtTile()` or `getPlantAt()` would be more consistent? I have a slight preference towards the former, for what it's worth.